src/sentry/snuba/query_subscription_consumer.py

```diff
@@ -200,13 +200,11 @@ def handle_message(self, message):
                     "timestamp": contents["timestamp"],
                     "query_subscription_id": contents["subscription_id"],
                     "project_id": subscription.project_id,
-                    "request": contents.get("request"),
                     "subscription_dataset": subscription.dataset,
                     "subscription_query": subscription.query,
                     "subscription_aggregation": subscription.aggregation,
                     "subscription_time_window": subscription.time_window,
                     "subscription_resolution": subscription.resolution,
-                    "contents": contents,
```

Comment:
This contains everything, so I decided to remove it too, in case the issue is related to the contents of `request`. We have all the fields we need specified explicitly, afaik